Content.Client/Administration/UI/Tabs/ObjectsTab/ObjectsTab.xaml.cs

```diff
@@ -10,20 +13,20 @@ namespace Content.Client.Administration.UI.Tabs.ObjectsTab;
 [GenerateTypedNameReferences]
 public sealed partial class ObjectsTab : Control
 {
-    [Dependency] private readonly EntityManager _entityManager = default!;
+    [Dependency] private readonly IEntityManager _entityManager = default!;
     [Dependency] private readonly IGameTiming _timing = default!;
 
     private readonly List<ObjectsTabEntry> _objects = new();
-    private List<ObjectsTabSelection> _selections = new();
+    private readonly List<ObjectsTabSelection> _selections = new();
+    private bool _ascending = false;  // Set to false for descending order by default
+    private ObjectsTabHeader.Header _headerClicked = ObjectsTabHeader.Header.ObjectName;
+    private readonly Color _altColor = Color.FromHex("#292B38");
+    private readonly Color _defaultColor = Color.FromHex("#2F2F3B");
 
-    public event Action<ObjectsTabEntry, GUIBoundKeyEventArgs>? OnEntryKeyBindDown;
+    public event Action<GUIBoundKeyEventArgs, ListData>? OnEntryKeyBindDown;
 
-    // Listen I could either have like 4 different event subscribers (for map / grid / station changes) and manage their lifetimes in AdminUIController
-    // OR
-    // I can do this.
```

Comment:
I was actually considering moving this to a refresh button and load on open rather than having the update every 2 seconds.  I have never needed real time listing of these ids but maybe there is a use case outside of admin that would need it?

Reply:
Fixed this now with refresh on tabchange  to the object tab and a button